### PR TITLE
fix: Fixes the health check cache.

### PR DESCRIPTION
**Pull request by bgrozev**

### CODE REVIEW

File: src/main/java/org/jitsi/videobridge/health/Health.java [hunk 1]
```diff
@@ -182,7 +182,7 @@ public static void check(Videobridge videobridge)
 
         if (cachedExceptionTimestamp > 0 &&
```
Comment (paweldomas) [inline src/main/java/org/jitsi/videobridge/health/Health.java]: will the code be simpler if you include `cachedException != null` as the first `if` condition ?

Comment (paweldomas) [reply src/main/java/org/jitsi/videobridge/health/Health.java]: No, actually not. It would not be correct

Comment (bgrozev) [reply src/main/java/org/jitsi/videobridge/health/Health.java]: No, I want to finish execution (either throw or return) regardless of the value of cachedException

Comment (paweldomas) [reply src/main/java/org/jitsi/videobridge/health/Health.java]: right

**Pull request merged**